cfme/infrastructure/virtual_machines.py

```diff
@@ -648,7 +648,16 @@ def active(self):
             title = getattr(self, self.parent_vm.provider.SNAPSHOT_TITLE)
             view = navigate_to(self.parent_vm, 'SnapshotsAll')
             root_item = view.tree.expand_path(self.parent_vm.name)
-            return has_child(view.tree, '{} (Active)'.format(title), root_item)
+            from cfme.infrastructure.provider.rhevm import RHEVMProvider
+
+            if self.parent_vm.provider.one_of(RHEVMProvider):
```

Comment:
REQUIRED: I think you need to add BZ here. otherwise all rhv snapshot related tests will fail when it is fixed.

Reply:
Can you elaborate please?

Reply:
@izapolsk I checked `5.9` and there's no such thing `'{} (Active)'.format(title)` for snapshots there as well. You only see a bunch of snapshots and `Active VM (Active)` which is the same as in RHEV itself.

I made this change only for RHEV provider.

Reply:
@digitronik I checked this test for `vsphere65-nested` and it's different to RHEV, that's why no change for it is required.

1. When there're no snapshots, it shows 0 for `vsphere65-nested` but 1 for RHEV as it always has `Active VM (Active)` <- that's exactly what is displayed in RHEV itself

2. Using `vsphere65-nested` it shows which snapshot is active

![image](https://user-images.githubusercontent.com/42433123/53570270-5dec9700-3b66-11e9-8128-86cb31351a82.png)

whereas for RHEV it's always `Active VM (Active)` <- how it's displayed in RHEV itself

Reply:
@lina-nikiforova just put one more condition with this check. so as this BZ on `rhv` fixed its should fallow older way like.
```
if BZ(1561618, forced_streams=['5.10']).blocks and self.parent_vm.provider.one_of(RHEVMProvider):
```

Reply:
@digitronik this BZ is about different issue that I can't reproduce in 5.10 and no snapshots are missing. 
What I am saying it's always `Active VM (Active)` for RHEV, so I don't understand why it will go older way depending on this BZ.